modules/internal/ChapelRange.chpl

```diff
@@ -1747,10 +1977,12 @@ module ChapelRange {
   
     return (U(3), U(1));
   }
-  
+
+  pragma "no doc"
   inline proc chpl__extendedEuclid(u:int(32), v:int(32))
   { return chpl__extendedEuclidHelper(u,v); }
-  
+
+  pragma "no doc"
```

Comment:
I think it'd be good to add an entry to the `modules/internal/fixInternalDocs.sh` script for Ranges and use the `removePrefixFunctions` to avoid having to no doc all these internal functions. Long term, chpldoc should know not to document things starting with `chpl_`